crates/bevy_text/src/text2d.rs

```diff
@@ -49,7 +49,9 @@ use bevy_window::{PrimaryWindow, Window};
 /// # use bevy_color::Color;
 /// # use bevy_color::palettes::basic::BLUE;
 /// # use bevy_ecs::world::World;
-/// # use bevy_text::{Font, JustifyText, Text2d, TextLayout, TextFont, TextColor};
+/// # use bevy_ecs::prelude::Component;
+/// # use bevy_hierarchy::{ChildBuild, BuildChildren};
```

Comment:
doesn't exist anymore

Reply:
Ugh, okay, thanks for the poke. I didn't realize this failed to merge.